### ISSUE TITLE
fix github workflow, way to specify input files

### ISSUE THREAD

**Issue by chrisroederucdenver** [chrisroederucdenver/CCDA_OMOP_by_Python#37]
tl;dr Add a -f flag to specify the input file and take the paths out of both prototype_2 programs.
Fix the github action workflow.

----------------------------------------------------------------------------

Currently the programs in prototype_2 (data_driven_parse and layer_datasets) contain a list of input filepaths.
Before yesterday the referenced files in the same repo under resources. Yesterday I pulled those out into their own repo CCDA-data, and then went to fix the github action...

the github action checks out CCDA-data, but has to do so in the same tree, not a parallel tree and this is where the first problem is. My thinking of the dev. environment is for parallel trees. So the action creates a link (ln -s) from CCDA-data/resources to a local resources, and the programs can still use resources.

but over in the dev world, CCDA_OMOP_by_Python has neither a local resources directory, nor a link to CCDA-data/resources and fails. 

The hack i just checked in changes the data paths from 'resources/CCDA_CCD_Amublatory_whatever.xml to ../CCDA-data/resources/CCDA_CCD_Ambulatory_whatever.xml so they are found. But this breaks the action.

We could go down the road of having the dev environment create the link like I do on the github action, but it might be better just to add argsparse (if not already there) and use the -f flag to specify the file to run, pulling the dependency of file location out.

It should also be noted that the file paths are flat and don't have the subdirectories they did before.

**Comment by chrisroederucdenver** [comment]
fixed with commit e65621a, d12c382 etc.
